### ISSUE TITLE
Largest Contentful Paint cannot compute on SVGs

### ISSUE THREAD

**Issue by lukeshiru** [GoogleChrome/lighthouse#11326]
#### Provide the steps to reproduce
1. Run LH on [lshi.ru](https://lshi.ru) (made in Preact: [source](https://github.com/lukeshiru/lukeshiru.me)) or [rightonhana.me](https://rightonhana.me) (made in React: [source](https://github.com/rightonhana/rightonhana.me)).
2. Check the output

#### What is the current behavior?

The **Largest Contentful Paint** sections gives this error:
```
Something went wrong with recording the trace over your page load. Please run Lighthouse again. (NO_LCP)
```

![](https://i.imgur.com/APaDnvF.png)

#### What is the expected behavior?

Get meaningful output in that section.

#### Environment Information
* Affected Channels: DevTools
* Chrome version: `85.0.4183.83 (Official Build) (64-bit)`
* Operating System: MacOS/Linux/Windows
* Private Mode, no extensions running.

**Related issues**
https://github.com/GoogleChrome/lighthouse/issues/11180
https://github.com/GoogleChrome/lighthouse/issues/10869
https://github.com/GoogleChrome/lighthouse/issues/11003

**Comment by patrickhulce** [comment]
Thanks for filing @lukeshiru! These were indeed unique from the related issues you linked and are a Chromium bug. I filed https://bugs.chromium.org/p/chromium/issues/detail?id=1122168 with a minimal repro 👍

**Comment by patrickhulce** [comment]
So reply on that crbug says that according to the spec, pages like this have no LCP. That's definitely suboptimal, so I poked the upstream issue https://github.com/WICG/element-timing/issues/47#issuecomment-699067245. We'll see.

**Comment by MatRivard** [comment]
Hi @patrickhulce, any update on this? Experiencing the same issue with SVG on my site's homepage when running Lighthouse audit.

**Comment by patrickhulce** [comment]
It's not in Lighthouse's hands @MatRivard  https://github.com/WICG/element-timing/issues/47#issuecomment-699067245 is the correct place to track updates.

**Comment by andreseduardop** [comment]
@lukeshiru,  @MatRivard, @patrickhulce, I experienced the same LCP error (due to SVG content) and I was able to figure it out.
I have only placed one image on [this site's homepage](https://shower.center/) -starting development, with some empty and animated `<div>` tags. :smile: 
The solution was to add a little text at the bottom of the viewport.
So, score 100 in Lighthouse.
I ruled out other possible sources of error, such as: css animation, elements outside the viewport, "opacty: 0.0" on startup, "animation: infinite"; etc.
